swap/src/bin/asb.rs

```diff
@@ -242,6 +248,31 @@ async fn main() -> Result<()> {
             let config_json = serde_json::to_string_pretty(&config)?;
             println!("{}", config_json);
         }
+        Command::Logs { path, redact } => {
+            // use provided directory of default one
+            let default_dir = system_data_dir()?.join("logs");
+            let logs_dir = path.unwrap_or(default_dir);
+
+            // go through each file and print it
+            let log_files = read_dir(&logs_dir)?;
+
+            for entry in log_files {
+                let file_name = entry?.path();
+                let file = File::open(&file_name).await?;
+                let buf_reader = BufReader::new(file);
+                let mut lines = buf_reader.lines();
+
+                println!("====reading log from file `{}`", &file_name.display());
+
+                // print each line, redacted if the flag is set
+                while let Some(line) = lines.next_line().await? {
+                    let line = if redact { common::redact(&line) } else { line };
+                    println!("{}", line);
+                }
+
+                println!("====no more log files found");
```

Comment:
The print statements probably more annoying than useful. Might be a pain when parsing?

Reply:
i removed the println's and added two tracing infos at the beginning instead so they won't interfere with the output. I also added an option for printing to a file directly.